script/bootstrap

```diff
@@ -54,11 +54,14 @@ install_packages_apt()
 
     # mDNS
     sudo apt-get install --no-install-recommends -y libavahi-client3 libavahi-common-dev libavahi-client-dev avahi-daemon
-    (MDNS_RESPONDER_SOURCE_NAME=mDNSResponder-878.30.4 \
+    (MDNS_RESPONDER_SOURCE_NAME=mDNSResponder-1310.80.1 \
         && cd /tmp \
         && wget -4 --no-check-certificate https://opensource.apple.com/tarballs/mDNSResponder/$MDNS_RESPONDER_SOURCE_NAME.tar.gz \
         && tar xvf $MDNS_RESPONDER_SOURCE_NAME.tar.gz \
-        && cd $MDNS_RESPONDER_SOURCE_NAME/mDNSPosix \
+        && cd /tmp/$MDNS_RESPONDER_SOURCE_NAME/Clients \
+        && sed -i '/#include <ctype.h>/a #include <stdarg.h>' dns-sd.c \
```

Comment:
We tried to upgrade mDNSResponder once but it's not compiling. 
Guess this is the fix.

Reply:
Yes. `stdarg.h` is used for `va_start` `va_end` and `sys/param.h` is used for `MIN`.